### PR TITLE
feat: Settings About

### PR DESCRIPTION
**Pull request by sdsantos**

### CODE REVIEW

File: app/src/main/java/tech/relaycorp/gateway/ui/settings/LicensesActivity.kt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package tech.relaycorp.gateway.ui.settings
+
+import android.os.Bundle
+import com.mikepenz.aboutlibraries.ui.LibsActivity
+import tech.relaycorp.gateway.R
+
+class LicensesActivity : LibsActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
```
Comment (gnarea) [inline app/src/main/java/tech/relaycorp/gateway/ui/settings/LicensesActivity.kt:8]: Can we limit the dependencies to runtime ones, excluding test and built-time dependencies?

**Review by gnarea — CHANGES_REQUESTED** [review]
There's a tinting issue on Android 5:

Screenshot_2020-10-12-14-51-20
Screenshot_2020-10-12-14-51-33

File: app/src/main/java/tech/relaycorp/gateway/ui/settings/LicensesActivity.kt (hunk 1 above)
Comment (sdsantos) [reply app/src/main/java/tech/relaycorp/gateway/ui/settings/LicensesActivity.kt:8]: Test dependencies only show up in `debug` builds, according to the documentation.

Comment (gnarea) [reply app/src/main/java/tech/relaycorp/gateway/ui/settings/LicensesActivity.kt:8]: Nice!

**Review by gnarea — DISMISSED** [review]

**Pull request closed without merging**